CHANGELOG.md

```diff
@@ -8,6 +8,40 @@
 > requests](https://github.com/fluxcd/toolkit/discussions) for Flux v2
 > and Toolkit components.
 
+## 1.20.2 (2020-08-19)
+
+This patch release includes a bump to kubectl, which may help users
+experiencing sync timeouts with CRDs.
+
+### Fixes
+
+- retain the error when appending the stderr output
+  [fluxcd/flux#3219][]
+- Fix some typos [fluxcd/flux#3257][]
+- Avoid spamming KMS when using SOPS decryption [fluxcd/flux#3248][]
```

Comment:
```suggestion
- Update SOPS to v3.6.0 and filter files before decryption [fluxcd/flux#3248][]
```

Reply:
I changed this one because I wanted people who were affected to understand it's a fix for them -- not because the PR git commit message is wrongly worded, just because a problem and its fix don't always obviously match up. I accept that my version may not be accurate or well-phrased though :-)